databases/week2/part 2.sql

```diff
@@ -0,0 +1,20 @@
+CREATE database lesson2_part2;
+use lesson2_part2;
+create table class(
+ id int unsigned primary key auto_increment, 
+ name varchar(225), 
+ begins datetime, 
+ ends datetime);
+ 
+ create table student( 
+ id int unsigned primary key auto_increment, 
+ name varchar(225), 
+ email varchar(255),
+ phone datetime, 
+ class_id int unsigned not null,
```

Comment:
The class_id is not an attribute of a student.
you need a table which connects the student to the classes which the student attend.
In this case the student will only have one class.